easier68k/core/opcodes/move.py

```diff
@@ -353,27 +383,25 @@ def from_binary(cls, data: bytearray) -> (str, int):
 
         # check opcode
         if opcode_bin != 0b00:
-            return (None, 0)
+            return None
 
         # the binary will contain the MoveSize, convert this to an OpSize used by everything else
         size = MoveSize(size_bin).to_op_size()
 
         # check size
         if size not in Move.valid_sizes:
             print('size not in valid', size)
-            return (None, 0)
+            return None
 
         wordsUsed = 1
 
         src_EA = parse_ea_from_binary(source_mode_bin, source_register_bin, size, True, data[wordsUsed*2:])
         wordsUsed += src_EA[1]
 
         dest_EA = parse_ea_from_binary(destination_mode_bin, destination_register_bin, size, False, data[wordsUsed*2:])
-        wordsUsed += dest_EA[1]
```

Comment:
What's the reasoning behind removing this but not the other one?

Reply:
`wordsUsed` isn't returned anymore, so updating its value after this statement has no purpose

Reply:
Ah, I see, okay.